src/auth/requireAuth/RequireAuth.jsx

```diff
@@ -33,7 +33,7 @@ const RequireAuth = () => {
 
         const config = {
           method: "post",
-          url: "https://team-worker.herokuapp.com/api/v1/auth/token/",
+          url: "https://teamwork.up.railway.app/api/v1/auth/token/",
```

Comment:
Given the fact that the baseurl keeps changing, i suggest we do not hardcode this in the project.
we can have it somewhere in the code as a variable and then reuse it in the files of interest

Reply:
Done. I set it as a variable in useAxios and then exported it from there